types/terser-webpack-plugin/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "dependencies": {
-    "terser": "^5.3.8"
+    "terser": "^5.3.8",
+    "webpack": "^5.0.0"
```

Comment:
`terser-webpack-plugin@5` specifies a [peer dependency for `webpack@^5.1.0`](https://github.com/webpack-contrib/terser-webpack-plugin/blob/e6798814cd64a74f888e2d241501fc08c8676ed8/package.json#L40-L42). Should this also use `^5.1.0` then?
```suggestion
    "webpack": "^5.1.0"
```

Reply:
I think so.

Reply:
@buschtoens I encountered a problem while using this package. 
<details>
<summary>
Error message
</summary>
<pre>
<code>
(alias) new TerserPlugin(opts?: TerserPlugin.TerserPluginOptions | undefined): TerserPlugin
import TerserPlugin
Excessive stack depth comparing types '{ mode: "production"; target: string; context: string; entry: { index: string[]; }; devtool: string; output: { path: string; publicPath: string; filename: string; chunkFilename: string; }; optimization: { ...; }; ... 4 more ...; plugins: WebpackPluginInstance[]; }' and 'Configuration'.ts(2321)
Type 'TerserPlugin' is not assignable to type '((this: Compiler, compiler: Compiler) => void) | WebpackPluginInstance | "..."'.
  Type 'TerserPlugin' is not assignable to type 'WebpackPluginInstance'.
    Types of property 'apply' are incompatible.
      Type '(compiler: import("/Users/tqma/Projects/github/react-torch/node_modules/@types/terser-webpack-plugin/node_modules/webpack/types").Compiler) => void' is not assignable to type '(compiler: import("/Users/tqma/Projects/github/react-torch/node_modules/webpack/types").Compiler) => void'.
        Types of parameters 'compiler' and 'compiler' are incompatible.
          Type 'import("/Users/tqma/Projects/github/react-torch/node_modules/webpack/types").Compiler' is not assignable to type 'import("/Users/tqma/Projects/github/react-torch/node_modules/@types/terser-webpack-plugin/node_modules/webpack/types").Compiler'.
            The types of 'hooks.shouldEmit' are incompatible between these types.
              Type 'import("/Users/tqma/Projects/github/react-torch/node_modules/tapable/tapable").SyncBailHook<[import("/Users/tqma/Projects/github/react-torch/node_modules/webpack/types").Compilation], boolean, import("/Users/tqma/Projects/github/react-torch/node_modules/tapable/tapable").UnsetAdditionalOptions>' is not assignable to type 'import("/Users/tqma/Projects/github/react-torch/node_modules/tapable/tapable").SyncBailHook<[import("/Users/tqma/Projects/github/react-torch/node_modules/@types/terser-webpack-plugin/node_modules/webpack/types").Compilation], boolean, import("/Users/tqma/Projects/github/react-torch/node_modules/tapable/tapable").Uns...'.ts(2322)
</code>
</pre>
</details>
This problem seems to be caused by this

![image](https://user-images.githubusercontent.com/22792671/102182298-fba18980-3ee6-11eb-96cf-49ce7f8c75bb.png)

It will be a webpack package in `/node_modules/@types/terser-webpack-plugin`. Whether the `"webpack": "^5.1.0"` line in dependences needed to remove.